### PR TITLE
Fix 2MASS-GC01 and 2MASS-GC02

### PR DESCRIPTION
**Pull request by GurrenLagannTSS**

### CODE REVIEW

**Review by ajtribick — DISMISSED** [review]

**Comment by 375gnu** [comment]
What the source of the data? http://spider.seds.org/spider/MWGC/2mass-gc01.html & http://spider.seds.org/spider/MWGC/2mass-gc02.html give other values.

**Comment by GurrenLagannTSS** [comment]
They're derived from the AppMag at http://people.smp.uq.edu.au/HolgerBaumgardt/globular/index.html with the extinctions manually applied.
But i feel like the values you supplied are likely more reliable in this case, so i can adopt those.

**Comment by GurrenLagannTSS** [comment]
Done. Just in case, is a commit squish needed in this case?

File: data/globulars.dsc [hunk 1]
```diff
@@ -1854,15 +1854,15 @@ Globular "NGC 6553:GCl 88:C 1806-259"
 	InfoURL "http://simbad.u-strasbg.fr/sim-id.pl?Ident=NGC 6553"
 }
 
-Globular "2MASS-GC02:[HJK2000] GC02"
+Globular "2MASS-GC02:Hurt 2: [HJK2000] GC02"
```
Comment (375gnu) [inline data/globulars.dsc:1857]:
```suggestion
Globular "2MASS-GC02:Hurt 2:[HJK2000] GC02"

```

**Review by 375gnu — APPROVED** [review]

**Comment by 375gnu** [comment]
Yes, you need to use "Squash and merge"

**Pull request merged**